README.Rmd

```diff
@@ -221,24 +221,90 @@ The residential model uses a variety of individual and aggregate features to det
 library(dplyr)
 library(tidyr)
 library(yaml)
+library(jsonlite)
+library(purrr)
+library(tibble)
+
+# Some values are derived in the model itself, so they are not documented
+# in the dbt DAG and need to be documented here
+hardcoded_descriptions <- tribble(
+  ~"column", ~"description",
+  "sale_year", "Sale year calculated as the number of years since 0 B.C.E",
+  "sale_day",
+  "Sale day calculated as the number of days since January 1st, 1997",
+  "sale_quarter_of_year", "Character encoding of quarter of year (Q1 - Q4)",
+  "sale_month_of_year", "Character encoding of month of year (Jan - Dec)",
+  "sale_day_of_year", "Numeric encoding of day of year (1 - 365)",
+  "sale_day_of_month", "Numeric encoding of day of month (1 - 31)",
+  "sale_day_of_week", "Numeric encoding of day of week (1 - 7)",
+  # nolint start: line_length_linter
+  "sale_post_covid", "Indicator for whether sale occurred after COVID-19 was widely publicized (around March 15, 2020)"
+  # nolint end
+)
```

Comment:
**nitpick:** I would just throw `# nolint start` around the entire block here then keep related things on the same line:

```suggestion
# in the dbt DAG and need to be documented here
# nolint start
hardcoded_descriptions <- tribble(
  ~"column", ~"description",
  "sale_year", "Sale year calculated as the number of years since 0 B.C.E",
  "sale_day", "Sale day calculated as the number of days since January 1st, 1997",
  "sale_quarter_of_year", "Character encoding of quarter of year (Q1 - Q4)",
  "sale_month_of_year", "Character encoding of month of year (Jan - Dec)",
  "sale_day_of_year", "Numeric encoding of day of year (1 - 365)",
  "sale_day_of_month", "Numeric encoding of day of month (1 - 31)",
  "sale_day_of_week", "Numeric encoding of day of week (1 - 7)",
  "sale_post_covid", "Indicator for whether sale occurred after COVID-19 was widely publicized (around March 15, 2020)"
)
# nolint end
```

Reply:
Smart, done in [233b7c3](https://github.com/ccao-data/model-res-avm/pull/18/commits/233b7c385a3d1bc3842fa16eb00650d5c9e776ed).